build_defs/antlr.bzl

```diff
@@ -35,12 +35,13 @@ def antlr4_cc_combined(name, src, listener = False, visitor = True):
         language = "Cpp",
         listener = listener,
         visitor = visitor,
-        package = "",
+        package = "auth_logic_cc_generator",
```

Comment:
You shouldn't hard code it to `auth_logic_cc_generator`. Simply remove `package = ""`. That should work I think.

Reply:
oh I thought doing this way, keeps the whole AuthLogicParser and AuthLogicLexer under the auth_logic_cc_generator namespace. Otherwise`using auth_logic_cc_generator::AuthLogicParser` in `ast_construction.cc` would fail.

Reply:
Removing package totally fails build on the above mentioned using statement

Reply:
`package = name,` should work and I think it is compatible with the internal behavior.